packages/json-rpc-engine/src/JsonRpcEngine.ts

```diff
@@ -9,21 +9,23 @@ import type {
   JsonRpcParams,
   PendingJsonRpcResponse,
 } from '@metamask/utils';
-import { hasProperty, isJsonRpcRequest } from '@metamask/utils';
+import {
+  hasProperty,
+  isJsonRpcNotification,
+  isJsonRpcRequest,
+} from '@metamask/utils';
 
 export type JsonRpcEngineCallbackError = Error | SerializedJsonRpcError | null;
```

Comment:
`JsonRpcEngineCallbackError` is no longer used anywhere in core, and only has two instances in mobile's `RPCMethodMiddleware.ts`.